OwenConferenciasApp/screens/NewEvents.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  ScrollView,
+  TextInput,
+  Alert,
+} from 'react-native';
+
+export default class NewEvent extends React.Component {
+  render() {
+    return (
+      <View style={styles.mainContainer}>
```

Comment:
I recommend to use `SafeAreaView` component on screen components
https://reactnative.dev/docs/safeareaview

Reply:
Same apply for all screen components